stetho-sample/src/main/java/com/facebook/stetho/sample/IRCConnectActivity.java

```diff
@@ -0,0 +1,77 @@
+package com.facebook.stetho.sample;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.facebook.stetho.sample.IRCChatActivity.IRCChatActivityResult;
+
+import java.util.Random;
+
+public class IRCConnectActivity extends Activity {
+  private static final String DEFAULT_HOST = "irc.freenode.net";
+
+  private static final int REQUEST_CODE_CHAT = 1;
+
+  private TextView mIRCPriorError;
+  private EditText mIRCServer;
+  private EditText mIRCNickname;
+
+  public static void show(Context context) {
+    Intent intent = new Intent(context, IRCConnectActivity.class);
+    context.startActivity(intent);
+  }
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.irc_connect_activity);
+
+    mIRCPriorError = (TextView) findViewById(R.id.irc_prior_error);
+    mIRCServer = (EditText) findViewById(R.id.irc_server);
+    if (TextUtils.isEmpty(mIRCServer.getText())) {
+      mIRCServer.setText(DEFAULT_HOST);
+    }
+    mIRCNickname = (EditText) findViewById(R.id.irc_nickname);
+    if (TextUtils.isEmpty(mIRCNickname.getText())) {
+      mIRCNickname.setText("stetho" + (new Random().nextInt(9999) + 1));
```

Comment:
Turns out the issue you were hitting appears to have been caused by the usage of "Guest".  The message we were getting was telling us that Guest usernames are special cased :)